### ISSUE TITLE
Mutable table incorrectly changed to immutable, when passed as `TABLES` parameter to function module call

### ISSUE THREAD

**Issue by bnichell** [SAP/abap-cleaner#209]
In function module calls `TABLES` parameter, is passed using a reference[^1].
Thus, in following example, the keyword `DATA` is incorrectly changed to `FINAL`, if option "Use FINAL for immutable variables" is active:

```ABAP
CLASS lcl_test DEFINITION.
  PUBLIC SECTION.
    TYPES: BEGIN OF test_structure,
             a TYPE string,
             b TYPE string,
           END OF test_structure.
    TYPES test_table TYPE TABLE OF test_structure WITH DEFAULT KEY.

    METHODS test_method.
ENDCLASS.



CLASS lcl_test IMPLEMENTATION.
  METHOD test_method.
    DATA(lt_string) = VALUE test_table( ).       " <<<< Variable needs to stay mutable
    CALL FUNCTION 'FUNCTION_MODULE_A'
      TABLES strings = lt_string.
  ENDMETHOD.
ENDCLASS.
```

[^1]: https://help.sap.com/doc/abapdocu_751_index_htm/7.51/en-us/abapcall_function_parameter.htm#!ABAP_ADDITION_3@3@

**Comment by jmgrassau** [comment]
Hi Benedikt, 

thanks a lot for reporting this bug! This will be fixed in the next release.

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Benedikt, 

this bug is now fixed with version 1.12.0, which was just released!

Kind regards,
Jörg-Michael
